models/canadarm/urdf/SSRMS_Canadarm2.urdf.xacro

```diff
@@ -0,0 +1,352 @@
+<?xml version="1.0" ?>
```

Comment:
Fine with this being added but the `models/canadarm/urdf/SSRMS_Canadarm2.urdf` should not be removed as canadarm_moveit_config depends on that. Note moveit doesn't recognize `package://` paths hence they were hardcoded.